src/main/resources/openapi/model/api.yaml

```
@@ -13,6 +13,46 @@ tags:
 #    description: "local host"
 
 paths:
+  /notification/sms/{refId}:
+    post:
```

Is this a post or a get?? since we are getting the sms notifications given the refId

```suggestion
    post: paths:
  /ping:
    get:
      responses:
        '200':
          description: OK
```

---

oooh my bad, i made a mistake

---

Please make the change and I will quickly approve your PR